content/yaml-publishing/google-play.md

```diff
@@ -10,39 +10,83 @@ Codemagic enables you to automatically publish your android application to Googl
 {{</notebox>}}
 
-Codemagic enables you to automatically publish your app either to one of the predefined tracks (`internal`- publish for internal testing and QA, `alpha`- publish for testing with a small group of trusted users, `beta`- publish for testing to a wider set of users and `production`- release the app to production) on Google Play or to your custom closed testing tracks. In order to do so, you will need to [set up a service account in Google Play Console](../knowledge-base/google-play-api/) and save the contents of the `JSON` key file as a [secure environment variable](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) in application or team settings.
+Codemagic enables you to automatically publish your app either to one of the predefined tracks on Google Play or to your custom closed testing tracks. 
 
-If your application supports [in-app updates](https://developer.android.com/guide/playcore/in-app-updates), Codemagic allows setting the update priority. Otherwise, `in_app_update_priority` can be omitted or set to `0`.
+In order to do so,a service account is required when setting up publishing to Google Play. The service account JSON key file must be added to Codemagic to authenticate with these services.
 
-In addition, Codemagic supports [staged releases](https://support.google.com/googleplay/android-developer/answer/6346149?hl=en), allowing users to choose which fraction of the testers or users get access to the application. To release to everyone, omit `rollout_fraction` from codemagic.yaml.
+## Setting up Google Play API access
 
-```yaml
-publishing:
-  google_play: # For Android app
-    credentials: $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS # Contents of the JSON key file for Google Play service account saved as a secure environment variable
-    track: alpha # Name of the track internal, alpha, beta, production, internal app sharing, or your custom track name
-    in_app_update_priority: 3 # Optional. Priority of the release (only set if in-app updates are supported): integer in range [0, 5]
-    rollout_fraction: 0.25 # Optional. Rollout fraction (set only if releasing to a fraction of users): value between (0, 1)
-    changes_not_sent_for_review: true # Optional boolean. To be used ONLY if your app cannot be sent for review automatically *
-    submit_as_draft: true # Optional boolean. Publish artifacts under a draft release. Can not be used together with rollout_fraction. Defaults to false
-```
+1. To allow Codemagic to publish applications to Google Play, it is necessary to set up access using Google Play API. 
+
+2. In Google Play Console, navigate to **Setup > API access** and click **Learn how to create service accounts**.<br><br>
+![Google play start](../uploads/s1.png). This will lead you to the Google Cloud Platform. 
```

Comment:
```suggestion
2. In the Google Play Console, navigate to **Setup > API access** and click **Learn how to create service accounts**.<br><br>
![Google play start](../uploads/s1.png). This will lead you to the Google Cloud Platform. 
```